.circleci/config.yml

```diff
@@ -65,21 +65,39 @@ commands:
           paths:
             - ./node_modules
 
-  app-build-and-store:
-    description: Run build and store archive of it
+  build:
```

Comment:
`build` is the new command name for deleted `build-app` command. With 2 new steps: `checkout-code` and `setup-shared-env`. This removes duplicated steps in several jobs that calls the `build` command.

`build-atcp-assets` and `execute-study-specific-commands` commands are removed and combined their steps into `build` command. Because, `execute-study-specific-commands` is used only by `build-atcp-assets`. And `build-atcp-assets` is used only by `build`. This change reduced code clutter in config file.